airflow/dags/ingestion.py

```diff
@@ -13,19 +13,45 @@
 data_ingestion_dag = DAG(
     'data_ingestion_dag',
     default_args=default_args,
+    schedule_interval="@weekly",  # Run once a week at midnight on Sunday morning
     description='The data ingestion pipeline.')
 
 
-def ingest_data_to_gcs(url: str, data: dict):
-    resp = requests.post(url, data)
-    if resp.status_code != 200:
-        raise Exception('Failed response code: {}'.format(resp.status_code))
+def create_gcs_ingest_operator(task_id: str, payload: dict) -> PythonOperator:
+    return create_request_operator(task_id, Variable.get("INGEST_TO_GCS_SERVICE_ENDPOINT"), payload)
 
 
-ingest_to_gcs_data_task = PythonOperator(
-    task_id='ingest_to_gcs',
-    python_callable=ingest_data_to_gcs,
-    op_kwargs={'url': Variable.get("INGEST_TO_GCS_SERVICE_ENDPOINT"), 'data': {
-        'hello': 'world'}},
-    dag=data_ingestion_dag,
-)
+def create_bq_ingest_operator(task_id: str, payload: dict) -> PythonOperator:
+    return create_request_operator(task_id, Variable.get("GCS_TO_BQ_SERVICE_ENDPOINT"), payload)
+
+
+def create_request_operator(task_id: str, url: str, payload: dict) -> PythonOperator:
+    return PythonOperator(
+        task_id=task_id,
+        python_callable=service_request,
+        op_kwargs={'url': url, 'data': payload},
+        dag=data_ingestion_dag,
+    )
+
+
+def service_request(url: str, data: dict):
+    try:
+        resp = requests.post(url, json=data)
+        resp.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        raise Exception('Failed response code: {}'.format(err))
+
+
+# CDC Covid Deaths
+cdc_covid_deaths_payload = {"message": {"is_airflow_run": True,
+                                        "filename": "cdc_deaths",
+                                        "gcs_bucket": Variable.get("CDC_COVID_DEATH_BUCKET"),
```

Comment:
I have a slight preference for using the variable name of the bucket directly instead of creating a variable name for the bucket for each source since it makes fewer things to keep track of. This assuming that each source will be sharing the same bucket.

Reply:
So you're thinking we should use a single bucket for all our data?

Reply:
Oh, I was under the impression that there would be one shared landing bucket for all the data sources since that was the model of the prototype. We don't necessarily need to do it like that, but just so I'm up to speed, could you explain the reasoning behind having a different landing bucket per data source?

Reply:
I see. I think I misinterpreted how we implemented the prototype. Until a need arises to separate them, I don't see a reason why not to keep them all in a single bucket.

Reply:
Updated this to use the same `gcs_landing_bucket` name as terraform.